modules/png-extraction/config.json

```diff
@@ -1,13 +1,13 @@
 {
-	"DICOMHome": "/Users/pradeeban/Downloads",
-	"OutputDirectory": "/Users/pradeeban/Downloads/root",
-	"Depth": 0,
-	"SplitIntoChunks": 1,
-	"PrintImages": true,
-	"CommonHeadersOnly": false,
-	"UseProcesses": 0,
-	"FlattenedToLevel": "patient",
-	"is16Bit":true,
-	"SendEmail": true,
-	"YourEmail": "test@test.test"
+    "DICOMHome": "E:/MyProjects/GSOC/niffler-ui/lol/imgs",
+    "OutputDirectory": "E:/MyProjects/GSOC/niffler-ui/lol/outputs",
+    "Depth": 1,
+    "SplitIntoChunks": 1,
```

Comment:
Please do not commit changes to this file. Remove this file from the pull request.

Reply:
ok will check that